zbus/src/lib.rs

```diff
@@ -59,7 +59,7 @@ pub type Address = address::Address;
 mod guid;
 pub use guid::*;
 
-mod message;
+pub mod message;
 pub use message::*;
```

Comment:
I think we should also provide deprecated export at the top-level for message mod hierarchy.

Reply:
ok

Reply:
This is not yet resolved. We're exposing `Message*` on top-level afaict.